spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+feature 'visit home page' do
+  before { @loans = create_list(:loan, 3, :active) }
+  context 'on the home page' do
+    before { visit root_path }
+
+    it 'shows active loans' do
+      active_loans = @loans.select{ |loan| loan.status == I18n.t(:loan_active) }
+      active_loans.each do |loan|
+        check_loan_content(loan)
+      end
+    end
+
+    context 'with both active and completed loans' do
+      before { @loans = create_list(:loan, 10) }
+
+      it 'shows completed loans on their tab' do
+        click_link 'Completed'
+        completed_loans = @loans.select{ |loan| loan.status == I18n.t(:loan_completed) }
+        completed_loans.each do |loan|
+          check_loan_content(loan)
+        end
+      end
+
+      it 'shows all loans on all tab' do
+        click_link 'All'
+        @loans.each do |loan|
+          check_loan_content(loan)
+        end
+      end
+    end
+
+    context 'with many loans' do
+      before { @loans = create_list(:loan, 25, :active) }
+      it 'paginates loan list' do
+        visit root_path
+        expect(page).to have_selector('div.pagination ul.pagination li.next a')
```

Comment:
It's checking this now, it's just on the line *after* the selector so github isn't picking up that it's related to your comment :smile_cat: